install_scripts/java.sh

```diff
@@ -0,0 +1,5 @@
+###
+# Java
+###
+
+apt-get -y install openjdk-7-jre
```

Comment:
you know, when i read the blazegraph installation guide initially (https://wiki.blazegraph.com/wiki/index.php/Installation_guide), it seems to call out Java 7 explicitly as a requirement. there was also something i read in the 2.0 release candidate notes that made me think 2.0 would be java 8 compatible, but i can't find that now.

but, looking at https://www.blazegraph.com/download/ i think it might actually be >= 7 after all. i'll give it a shot and switch to 8 if things seem to work.